docs/infrastructure/sbt.md

```diff
@@ -122,26 +120,17 @@ information is used by `enso --version`.
 Truffle annotation processor generates a file that registers instruments
 provided by the runtime. Unfortunately, with incremental compilation, only the
 changed instruments are recompiled and the annotation processor does not detect
-this, so un-changed instruments get un-registered.
-
-To fix this, the pre-compile task defined in
-[`FixInstrumentsGeneration`](../../project/FixInstrumentsGeneration.scala)
-detects changes to instruments and if only one of them should be recompiled, it
-forces recompilation of all of them, to ensure consistency.
-
-For unclear reasons, if this task is attached to
-`Compile / compile / compileInputs`, while it runs strictly _before_
-compilation, the deleted class files are not always all recompiled. So instead,
-it is attached directly to `Compile / compile`. This technically could allow for
-a data race between this task and the actual compilation that happens in
-`compileIncremental`, but in practice it seems to be a stable solution.
-
-Sometimes it is unable to detect the need for recompilation before it takes
-place. To help that, there is another task that replaces the default `compile`
-task, which executes the default compilation task and after it, verifies the
-consistency of instruments files. As it cannot restart compilation, to preserve
-consistency it ensures the instruments will be recompiled the next time and
-stops the current compilation task, asking the user to restart it.
+this, so un-changed instruments get overwritten.
+
+In the past we had a pre-compile task (defined in
+[`FixInstrumentsGeneration`](../../project/FixInstrumentsGeneration.scala)) that
+detected changes to instruments and if only one of them was to be recompiled, it
```

Comment:
This link is not valid anymore. I'd suggest removing the link.

Reply:
Mostly left it for historical reasons, but sure, can remove it

Reply:
I think the description can stay. I'd just remove the link because it doesn't work anyway so it may be confusing (why the link doesn't work). You may change it to a GitHub link to some old commit on develop.